src/patternfly/_base.scss

```diff
@@ -153,3 +163,12 @@ html {
     }
   }
 } // ends if
+
+// remove scrollbar space
+::-webkit-scrollbar {
```

Comment:
`_base.scss` is the base stylesheet for PF4. When this stylesheet is added to a site, the styles in it will apply to everything on the site. For example, the workspace no longer has a scrollbar. All buttons (unless there is specific CSS for the button) will no longer have padding, bbackground, border, etc. I know you just moved these from the sequence stylesheet, but did you mean to limit the scope of these changes to the sequence component?

Reply:
Yes, but move them inside of the `.pf-c-sequence { // in here somewhere }` selector block. I believe before you had added them outside of that block, which will apply to the entire site (similar to how adding it to `_base.scss` applies to the whole site). And if the styles for `button` are for the sequence control element, apply the CSS to `.pf-c-sequence__control`, not `button`

Reply:
thanks for clarifying that - moved them to their respective areas